docs/general/naming.md

```diff
@@ -288,14 +288,19 @@ Examples:
 - `system.paging.faults`, `system.disk.operations`, and `system.network.packets`
   should be pluralized, even if only a single data point is recorded.
 
-#### Use `count` instead of pluralization for UpDownCounters
+#### Do not pluralize UpDownCounter names
 
-If the value being recorded represents the count of concepts signified
-by the namespace then the metric should be named `count` (within its namespace).
+UpDownCounter names MUST NOT be pluralized.
+
+If the value being recorded represents a count of the concepts defined by the namespace,
+a suffix such as `count`, `active`, etc. SHOULD be used within that namespace.
```

Comment:
do we really need this guidance? we have `.count`, `.active`, `.used`, `.used_after_last_gc`, `.ready`, `.queued`, `.processed`, ....

I don't think we need to be prescriptive on which adjective to use. 

```suggestion
```